open_alchemy/build/__init__.py

```diff
@@ -1,8 +1,17 @@
 """Build a package with the OpenAlchemy models."""
-
+from enum import auto
```

Comment:
The pattern elsewhere is to import `enum` so that it can be used as a namespace and to reduce the number of imports. Do you think that makes sense here?

Reply:
Totally! As I like to follow the OpenStack coding guidelines, it is also mentioned there: https://docs.openstack.org/charm-guide/latest/coding-guidelines.html#import-style.

As a side note, maybe it would be worth creating a CONTRIBUTING.md file and add the "rules" you like contributors to follow? As an example, here is such a file that I like to include in my projects: https://github.com/scrapd/scrapd/blob/master/.github/CONTRIBUTING.rst. If you think that could benefit your project, I could create an issue where we could discuss the content.

Reply:
Very good idea, I will add that as a ticket as well.